portal/templates/redesign/help-and-support_new.html

```diff
@@ -61,26 +61,57 @@ <h1 class="banner__text--primary">Help and support</h1>
     <div class="background background--primary">
         <div class="col-sm-8 col-center">
             <h3>Contact us</h3>
-            <form>
+
+            <form method='post'>
+
+                {% csrf_token %}
+
+                <div class="row">
+                    <div class="col-sm-12">
+                        {{ form.non_field_errors }}
+                    </div>
+                </div>
                 <div class="row">
                     <div class="col-sm-6">
-                        <p class="small-text">Your name</p>
-                        <input type="text" placeholder="Firstname Lastname">
+                        <label class="small-text">Your name</label>
+                        {{ form.name.errors }}
+                        {{ form.name }}
```

Comment:
To be consistent with the rest of the redesign, could you please put the error messages right below the form fields? :)